### PR TITLE
feat: add more messages

### PR DESCRIPTION
**Pull request by idandagan1**
Added:
- empty response
- hebrew only
- error handling

### CODE REVIEW

File: src/components/search.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export function Search({setData}: {setData: (data: PersonData[]) => void}) {
     <Input
       dir='rtl'
       type="search"
-      placeholder="שם / שם משפחה / טלפון ..."
+      placeholder="שם פרטי/משפחה (בעברית) / טלפון ..."
```
Comment (vithano) [inline src/components/search.tsx:42]: can remove the phone part

File: src/components/searchResults.tsx [hunk 2]
```diff
@@ -3,7 +3,8 @@ import PersonData from "../app/utils/types";
 import {PersonCard} from "./person-card";
 
 export function SearchResults({data}: {data: PersonData[]}) {
-  if(data.length == 0) return null;
+  if (data.length == 0) return "לא נמצאו תוצאות";
```
Comment (vithano) [inline src/components/searchResults.tsx:6]: if the input is empty it should not show anything

Comment (idandagan1) [reply src/components/searchResults.tsx:6]: ok right. on it

**Comment by vithano** [comment]
good work , should also be added to onInputChange when the input is empty
setMessage('')

File: src/components/searchResults.tsx [hunk 3]
```diff
@@ -2,9 +2,12 @@
 import PersonData from "../app/utils/types";
 import {PersonCard} from "./person-card";
 
-export function SearchResults({data}: {data: PersonData[]}) {
-  if(data.length == 0) return null;
-  return (
+export function SearchResults({data, message}: {data: PersonData[], message: string}) {
+
+  return data.length === 0 && message ?
```
Comment (vithano) [inline src/components/searchResults.tsx:7]: i think the data.length === 0 is redundant? if we have a message lets show it, if not show the results

Comment (idandagan1) [reply src/components/searchResults.tsx:7]: yeah you're right, but for readiness you will know that the message will be displayed only on empty result scenarios.

Comment (vithano) [reply src/components/searchResults.tsx:7]: ok i can accept that, thanks for the help!

**Pull request merged**